### PR TITLE
Match EnFhgFire_Draw

### PR DESCRIPTION
**Pull request by fig02**
fix found by maide

### CODE REVIEW

File: src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c [hunk 1]
```diff
@@ -462,71 +462,70 @@ void EnFhgFire_Update(Actor* thisx, GlobalContext* globalCtx) {
...
+                gDPSetEnvColor(gfxCtx->polyXlu.p++, 165, 255, 75, 0);
+            }
 
-        gDPPipeSync(gfxCtx->polyXlu.p++);
+            gDPPipeSync(gfxCtx->polyXlu.p++);
 
-        Matrix_RotateZ(((f32)thisx->shape.rot.z / 32768.0f) * M_PI, 1);
-        gSPMatrix(gfxCtx->polyXlu.p++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_en_fhg_fire.c", 1801),
-                  G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            Matrix_RotateZ((thisx->shape.rot.z / 32768.0f) * 3.1416f, 1);
```
Comment (mzxrules) [inline src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c:497]: are you sure M_PI doesn't work here

Comment (fig02) [reply src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c:497]: yes

Comment (fig02) [reply src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c:497]: M_PI = 40490FDB
this is 40490FF9

File: src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c [hunk 2]
```diff
@@ -462,71 +462,70 @@ void EnFhgFire_Update(Actor* thisx, GlobalContext* globalCtx) {
...
-        func_800D1FD4(&globalCtx->mf_11DA0);
-        func_80093D84(globalCtx->state.gfxCtx);
-
-        gDPSetPrimColor(gfxCtx->polyXlu.p++, 0, 0, 0xFF, 0xFF, 0xFF, (s32)this->unk_160 & 0xFF);
-
-        if (this->fireMode > 0) {
-            gDPSetEnvColor(gfxCtx->polyXlu.p++, 0, 255, 255, 0);
-        } else {
+    {
+        GraphicsContext* gfxCtx = globalCtx->state.gfxCtx;
```
Comment (Roman971) [inline src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c:470]: Is the macro block really needed here?

Comment (fig02) [reply src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c:470]: @Kelebek1

Comment (fig02) [reply src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c:470]: based on maides other prs i think its something hes just doing everywhere

Comment (fig02) [reply src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c]: took it out

**Pull request merged**